client/features/auth-pages/SignInPage.tsx

```diff
@@ -25,84 +22,71 @@ type FormData = z.infer<typeof userSignInSchema>;
 const SignInPage = () => {
   useDocumentTitle('Sign In');
   const navigate = useNavigate();
+  const queryClient = useQueryClient();
   const googleAuthMutation = useGoogleAuthMutation();
   const githubAuthMutation = useGithubAuthMutation();
   const signInWithEmailAndPasswordMutation =
     useSignInWithEmailAndPasswordMutation();
-  const { data: userProfile } = useFetchUserProfile();
   const {
     register,
     handleSubmit,
     formState: { errors },
-  } = useForm<FormData>({ resolver: zodResolver(userSignInSchema) });
-  const createUserProfileMutation = useCreateUserProfileMutation();
-  const queryClient = useQueryClient();
-  const { isAuthenticated } = useAuth();
+  } = useForm<FormData>({
+    resolver: zodResolver(userSignInSchema),
+  });
   const { showNotification } = useNotification();
 
-  const handleAuthSuccess = async (name: string) => {
-    if (!userProfile) {
-      try {
-        const response = await createUserProfileMutation.mutateAsync({ name });
-        await queryClient.setQueryData(['userProfile'], response);
-        showNotification("Welcome back! You're now signed in.", 'success');
-        navigate('/generate');
-      } catch (error) {
-        console.error('Failed to create user profile:', error);
-      }
-    }
-  };
-
   const handleGoogleAuthClick = async () => {
     try {
-      await googleAuthMutation.mutateAsync();
-      await handleAuthSuccess('');
+      await googleAuthMutation.mutateAsync().then(() => {
```

Comment:
Just leaving a comment here to remind you to create a new ticket to address the issue with Google and Github Oauth users currently being able to log in to the app without a userProfile.